baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,233 @@
+import logging
```

Comment:
ah, oops, the `from __future__` things all the other files have should be in here too. it keeps things more py3-friendly.

i think the module docstring goes at the very very top too.